packages/webview/src/component/PodsList.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+import { getContext, onDestroy, onMount } from 'svelte';
+import { States } from '/@/state/states';
+import { NavPage } from '@podman-desktop/ui-svelte';
+import type { Unsubscriber } from 'svelte/store';
+import type { ContextResourceItems } from '/@common/model/context-resources-items';
+
+const updateResource = getContext<States>(States).stateUpdateResourceInfoUI;
+const currentContext = getContext<States>(States).stateCurrentContextInfoUI;
+
+let unsubscriber: Unsubscriber | undefined;
+
+$effect(() => {
+  if (currentContext.data?.contextName) {
+    subscribeToContext(currentContext.data.contextName);
+  } else {
+    unsubscriber?.();
```

Comment:
wondering if instead of calling unsuscriber?. it could call unsubscribe() method and this method call unsubscriber


IMHO it's quite hard to follow also the difference between onMount that does currentContext.subscribe() and the method subscribeToContext that is calling first unsubscribe also